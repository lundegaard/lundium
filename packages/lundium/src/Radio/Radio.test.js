import React from 'react';
import { shallow } from 'enzyme';

import Radio from './Radio';

describe('<Radio />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<Radio {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
