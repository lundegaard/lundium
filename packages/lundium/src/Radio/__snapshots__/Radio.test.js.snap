// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> matches snapshot when shallowly rendered 1`] = `
<forwardRef(FormField)
  control={[Function]}
  kind="radio"
/>
`;
